entrypoint-4.1.x.sh

```diff
@@ -42,6 +42,29 @@ then
   echo "done"
 fi
 
+# Import a cert from a possibly mounted kubernetes secret at /cert
```

Comment:
I noticed that 4.1.5 has builtin support for HTTPS/SSL certificates

![Bildschirmfoto vom 2020-07-15 15-00-50](https://user-images.githubusercontent.com/3609611/87548054-430fb980-c6ac-11ea-96a6-c6942118cee3.png)

Is this conflicting with this manual approach?

Reply:
I added a note in the README.  It doesn't look like it would cause a conflict but the main issue is that the certificates would not persist long term between containers when updating.

Reply:
Oh well actually, I just tested with a JKS and it actually persisted between versions.  It must be storing the certificate in mongodb.  I haven't actually tested the certificate installation by bind mounting them in.  I'll have to test that.

Reply:
If this is now working ootb and directly supported by the controller, I would probably remove the custom support via the `/cert` volume. Might be a bit confusing to have two ways to do the same thing. 
WDYT?

Reply:
OK, so bind mounting of the certs method worked as expected after I fixed the path to the `keytool` command.

Reply:
My initial thoughts were to remove the `/cert` volume feature but after thinking about it a bit more, the nice thing about using `/certs` is that it is automated.  In my case, I use letsencrypt certs so I am regularly renewing them every 90 days so having an easily automated way to manage them is deal and the `/cert` volume allows that to work just by updating the cert files and restarting the container as it will re-import them for me.  I currently don't use that - I actually have an nginx reverse proxy in front that terminates my own TLS certs but I was actually considering switching to using the `/cert` volume to remove a layer of complexity with nginx.